addon/services/current-user.js

```diff
@@ -11,7 +28,13 @@ export default Ember.Service.extend({
             return null;
         }
     }),
+    /**
+     * Fetch information about the currently logged in user. If no user is logged in, this method returns a rejected promise.
+     * @method load
+     * @return {Ember.RSVP.Promise}
```

Comment:
We should try to be consistent with how we reference `Promise` vs. `Ember.RSVP.Promise`